config/config_test.go

```diff
@@ -374,7 +374,7 @@ func TestMarshalSecretURL(t *testing.T) {
 	// u003c -> "<"
 	// u003e -> ">"
 	require.Equal(t, "\"\\u003csecret\\u003e\"", string(c), "SecretURL not properly elided in JSON.")
-	// Check that the marshaled data can be unmarshaled again.
+	// Check that the marshaled data can be unmarshalled again.
```

Comment:
Both are valid, sticking to one is preferred and the Go documentation uses `marshaled`.

Reply:
I've updated a branch `spelling-marshal-x`: https://github.com/jsoref/alertmanager/compare/spelling..jsoref:spelling-marshal-x -- If that's what you want, I'll force push.